pyccel/codegen/printing/pycode.py

```diff
@@ -458,6 +461,14 @@ def _print_NumpyRandint(self, expr):
             args += ", size = {}".format(size)
         return "randint({})".format(args)
 
+    def _print_NumpyNorm(self, expr):
+        type_name = type(expr).__name__
+        name = type_name[5:].lower()
+        return  "{name}({arg},axis={axis})".format(
+                name = name,
+                arg  = self._print(expr.python_arg),
+                axis=self._print(expr.axis))
```

Comment:
If `expr.axis=None` then we don't want to print the argument (see fcode.py for an example)